### PR TITLE
Final touches

### PR DESCRIPTION
**Pull request by MostafaShahat2021**
### In this project:
- ⭐️ Create a new game by using Leaderboard API.
- ⭐️ Add Submit function.
- ⭐️ Add Refresh function.
- ⭐️Improve the App's styling.

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -7,29 +7,35 @@
...
       </ul>
     </div>
-
     <div class="form">
       <h2>Add your score</h2>
       <input type="text" id="name" name="name" placeholder="Enter your name" required>
       <input type="number" id="score" name="score" placeholder="Enter your score" required>
       <button id="submit">Submit</button>
     </div>
   </main>
```
Comment (Shedrack-Sunday) [inline dist/index.html:37]: - [ ] You did have a beautiful design 👏 . However, please adjust the submit button and the position of the form to look exactly like the layout given on the wireframe to complement the good work you've done so far 👍 
- [ ] Though you might be of the opinion that the current design is "great", the suggestion to stick to requirements is a simulation of what your hiring manager and potential clients will ask for. So please check the image comparison below and adjust it appropriately 

## Wireframe:
image

## Your view:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @MostafaShahat2021    👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 



Highlights
✅All linters are checked
✅ Proper PR summary 
  
# Required Changes ♻️

- [x] I must admit you have really put in the effort so far in this project, however, the changes required have to do with a bit of styling.

_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: dist/index.html (hunk 1 above)
Comment (MostafaShahat2021) [reply dist/index.html:37]: Hi 👋 @Shedrack-Sunday , 
Hope all is well with you ☺️,
Thanks for your efforts in reviewing my code,
I'll adjust the submit button and the position of the form, as you asked  :+1: .
Best Regards

Comment (Shedrack-Sunday) [reply dist/index.html:37]: You're welcome @MostafaShahat2021🤗

**Review by TuyishimireEric — APPROVED** [review]
# Status: APPROVED 🟢  
### Hi  @MostafaShahat2021  👋 👋 

#### I'm @TuyishimireEric, Your code reviewer,
Good job on implementing the project requirements! 🥇 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it 🦾  
Congratulations! 🎉

## Highlights
✔ Awesome design👍 

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/enus/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MostafaShahat2021** [comment]
Hi 👋 @TuyishimireEric, Thanks a lot 💯 for your efforts in reviewing my code 🤗.

**Pull request merged**